adobecom/milo
# MWPW-142894: Active element accessibility

## Description
This PR adds a fix for the global navigation active element accessibility.

## Related Issue
Resolves: [MWPW-140281](https://jira.corp.adobe.com/browse/MWPW-140281)

## Testing instructions
1. activate VoiceOver;
2. go to the before MILO test URL and select the "Active element" link. You should hear: "Active element, group";
3. go to the after MILO test URL and select the "Active element" link. You should hear: "Dimmed current page, link, Active element";

## Test URLs
**Acrobat:**
- Before: https://www.stage.adobe.com/acrobat/online/sign-pdf.html?martech=off
- After: https://www.stage.adobe.com/acrobat/online/sign-pdf.html?martech=off&milolibs=mwpw-140281-a11y--milo--robert-bogos

**BACOM:**
- Before: https://main--bacom--adobecom.hlx.page/products/real-time-customer-data-platform/rtcdp?martech=off
- After: https://main--bacom--adobecom.hlx.page/products/real-time-customer-data-platform/rtcdp?martech=off&milolibs=mwpw-140281-a11y--milo--robert-bogos

**CC:**
- Before: https://main--cc--adobecom.hlx.page/?martech=off
- After: https://main--cc--adobecom.hlx.page/?martech=off&milolibs=mwpw-140281-a11y--milo--robert-bogos

**Homepage:**
- Before: https://main--homepage--adobecom.hlx.page/homepage/index-loggedout?martech=off
- After: https://main--homepage--adobecom.hlx.page/homepage/index-loggedout?martech=off&milolibs=mwpw-140281-a11y--milo--robert-bogos

**Blog:**
- Before: https://main--blog--adobecom.hlx.page/
- After: https://main--blog--adobecom.hlx.page/?milolibs=mwpw-140281-a11y--milo--robert-bogos

**Milo:**
- Before: https://main--milo--robert-bogos.hlx.page/drafts/rbogos/page-active-element?martech=off&georouting=off
- After: https://mwpw-140281-a11y--milo--robert-bogos.hlx.page/drafts/rbogos/page-active-element?martech=off&georouting=off
